d2l-slot-resize-observer.js

```diff
@@ -0,0 +1,81 @@
+import '@polymer/polymer/polymer-legacy.js';
+
+window.D2L = window.D2L || {};
+window.D2L.PolymerBehaviors = window.D2L.PolymerBehaviors || {};
+window.D2L.PolymerBehaviors.Table = window.D2L.PolymerBehaviors.Table || {};
+
+/** @polymerBehavior */
+D2L.PolymerBehaviors.Table.SlotResizeObserver = {
+
+	properties: {
+		_destructor: Function
+	},
+
+	detached: function() {
+		if (this._destructor) {
+			this._destructor();
+		}
+	},
+
+	_initSlotResizeObserver: function(slotWrapper, slot) {
+		if (this._destructor) {
+			this._destructor();
+		}
+
+		if (window.ResizeObserver && window.ResizeObserver.toString().indexOf('[native code') >= 0) {
+			// Use native ResizeObserver
+			var observer = new window.ResizeObserver(function() {
+				this._onSlotSizeChanged(slotWrapper);
+			}.bind(this));
+			observer.observe(slotWrapper);
+			this._destructor = function() {
+				observer.unobserve(slotWrapper);
+			};
+		} else {
+			// Use workaround
+			var lastWidth = slotWrapper.offsetWidth;

```

Comment:
An alternative may be to use `getBoundingClientRect`, which would yield a result closer to what the real ResizeObserver is intended to provide to observers.  Given that this restricted to table, not a huge deal to match it.

Reply:
I'm going to try experimenting with making a `<resize-aware>` Polymer 3 component that can be used elsewhere, but I'm not sure if it will work with nested slots or not. If it does, I'll use ``getBoundingClientRect` to make it behave like the ResizeObserver does